run_service.sh

```diff
@@ -680,15 +752,21 @@ else
     cd $service_dir
     # Build the image
     poetry run autonomy build-image
-    cp ../../$keys_json_path $keys_json
 fi
 
 # Build the deployment with a single agent
-poetry run autonomy deploy build --n $n_agents -ltm
+poetry run autonomy deploy build "../../$keys_json_path" --n $n_agents -ltm $password_argument
 
 cd ..
 
 add_volume_to_service "$PWD/trader_service/abci_build/docker-compose.yaml" "trader_abci_0" "/data" "$PWD/../.trader_runner/"
 
 # Run the deployment
-poetry run autonomy deploy run --build-dir $directory --detach
+poetry run autonomy deploy run --build-dir $directory --detach || true
```

Comment:
true is needed to proceed to clear the password stored by the framework, even if this command fails.

Reply:
Not needed anymore.